app/src/main/res/values/strings.xml

```diff
@@ -825,4 +825,12 @@
     <string name="tab_counter_content_description_one_tab">1 tab</string>
     <!-- Content description of the tab counter toolbar button when multiple tabs are open. First parameter will be replaced with the number of tabs (always more than one) -->
     <string name="tab_counter_content_description_multi_tab">%d tabs</string>
+
+    <!-- Browser long press popup menu -->
+    <!-- Copy the current url -->
+    <string name="browser_toolbar_long_press_popup_copy">Copy</string>
+    <!-- Paste & go the text in the clipboard -->
+    <string name="browser_toolbar_long_press_popup_paste_and_go">Paste &#038; Go</string>
```

Comment:
Can we comment what `&#038;` is to help the localization team? :)

Reply:
Or maybe we can use `&amp;`?

Reply:
Good call, @jyeontaek. That's a little more clear. Also added a comment! 😄